ddtrace/llmobs/_evaluations/ragas/faithfulness/evaluator.py

```diff
@@ -0,0 +1,85 @@
+import atexit
+from concurrent import futures
+import math
+import time
+from typing import Dict
+from typing import Optional
+
+from ddtrace import config
+from ddtrace.internal import forksafe
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.periodic import PeriodicService
+
+from ...._writer import LLMObsEvaluationMetricEvent
+
+
+logger = get_logger(__name__)
+
+
+class RagasFaithfulnessEvaluator(PeriodicService):
+    """Base class for evaluating LLM Observability span events"""
+
+    def __init__(self, interval: float, _evaluation_metric_writer=None, _llmobs_instance=None):
+        super(RagasFaithfulnessEvaluator, self).__init__(interval=interval)
+        self.llmobs_instance = _llmobs_instance
```

Comment:
Why do we need a reference to LLMObs?

Reply:
In the full implementation, we want to use the llmobs service to trace the ragas evaluations. However, we cannot directly LLMObs since it will result in a circular import. Since it's un-used right now I can also just remove it for this PR